### PR TITLE
Cleanup/asset compiling

### PR DESCRIPTION
**Pull request by eric-michel**

### CODE REVIEW

File: src/sass/emma-woocommerce.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/*--------------------------------------------------------------
+# Emma - Theme Styles
```
Comment (dswebsme) [inline src/sass/emma-woocommerce.scss]: This comment should be `Emma - WooCommerce Styles`

File: src/woocommerce.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*--------------------------------------------------------------
+# Emma - Webpack Admin Entrypoint
```
Comment (dswebsme) [inline src/woocommerce.js]: This comment should be `Emma - Webpack WooCommerce Entrypoint`

**Review by dswebsme — CHANGES_REQUESTED** [review]

File: src/sass/emma-woocommerce.scss (hunk 1 above)
Comment (eric-michel) [reply src/sass/emma-woocommerce.scss]: handled in d8bc22e61094f0ed3da2e85fa890fbfb6b504fd6

File: src/woocommerce.js (hunk 2 above)
Comment (eric-michel) [reply src/woocommerce.js]: handled in d8bc22e61094f0ed3da2e85fa890fbfb6b504fd6

**Pull request merged**